dotcom-rendering/scripts/webpack/webpack.config.server.js

```diff
@@ -39,6 +39,7 @@ module.exports = ({ sessionId }) => ({
 		runtimeChunk: false,
 	},
 	externals: [
+		// @ts-expect-error - webpack-node-externals types are incorrect
```

Comment:
Is there a link to an issue or PR that describes the problem with these types that we could include here for future reference?